snyk-container-scan-policy-ts/README.md

```diff
@@ -0,0 +1,71 @@
+# snyn-container-scan-policy
+
+Scan Pulumi-managed Docker containers with Snyk and Pulumi Policy as Code:
+
+- The code in the `infra` directory creates two `docker.Image` resources:
+    1. An image sourced from the `alpine` image, which does not have critical vulnerabilities.
+    1. An image sourced from the `debian` image, which has critical vulnerabilities.
+- The code in the `policy` directory contains a Pulumi Policy Pack which calls the Snyk CLI. If the Snyk CLI fails (e.g. because it detects vulnerabilities), the resource will be considered in violation and the `pulumi preview` (or `pulumi up`) operation will fail.
+
+To run the demo:
+
+```bash
+cd infra
+pulumi preview --policy-pack ../policy
```

Comment:
We should probably mention here:

* Ensure the Docker deamon is running
* `npm install && npm -C ../policy install`
* `pulumi stack init dev` (or whatever)